src/plugin-slots/StudioFooterHelpSectionSlot/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { PluginSlot } from '@openedx/frontend-plugin-framework';
+import HelpSection from '../../components/studio-footer/help-components/HelpSection';
+
+const StudioFooterHelpSectionSlot = () => (
+  <PluginSlot id="org.openedx.frontend.layout.studio_footer_help_section.v1" idAliases={['studio_footer_help_section_slot']}>
```

Comment:
Since this is a new slot it shouldn't have `idAliases`

```suggestion
const StudioFooterHelpSectionSlot = () => (
  <PluginSlot id="org.openedx.frontend.layout.studio_footer_help_section.v1">
```